policyengine_us/variables/gov/states/oh/tax/income/credits/oh_lump_sum_retirement_credit.py

```diff
@@ -0,0 +1,48 @@
+from policyengine_us.model_api import *
+
+
+class oh_lump_sum_retirement_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Ohio Lump Sum Retirement Income Credit"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://codes.ohio.gov/ohio-revised-code/section-5747.055"  # (C, D, E)
+    defined_for = "oh_retirement_income_credit_eligible"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.oh.tax.income.credits
+
+        person = tax_unit.members
```

Comment:
set a limit for age >= 65
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/a14c2aba-c63c-483a-a9f3-701233bc7d61)